cluster/sync.sh

```diff
@@ -3,15 +3,9 @@
 registry_port=$(./cluster/cli.sh ports registry | tr -d '\r')
 registry=localhost:$registry_port
 
-IMAGE_REGISTRY=${registry} make manager push-manager
+IMAGE_REGISTRY=${registry} make push-manager
```

Comment:
do we need this with --up-local?

Reply:
Let's remove it and see if we need it (daemon sets?)

Reply:
Agreed, we could, to make this PR smaller.